src/main/scala/nl/knaw/dans/easy/export/package.scala

```diff
@@ -25,12 +25,17 @@ import scala.xml._
 
 package object export {
 
-  def invert[T1,T2] (m: Map[T1,T2]): Map[T2,T1] =
-    m.map{case (key,value) => (value,key)}
+  def invert[T1, T2](m: Map[T1, T2]): Map[T2, T1] =
+    m.map { case (key, value) => (value, key) }
 
   def toSdoName(objectId: String): String =
     objectId.replaceAll("[^0-9a-zA-Z]", "_")
 
+  implicit class RichNode(n: Node) {
+    def headOfAttr(namespace: String, attrName: String): Option[String] =
+      n.attribute(namespace, attrName).flatMap(xs => xs.headOption.map(_.text))
+  }
+
```

Comment:
A word of caution: don't make the convenience of things in the IDE too high a priority.